tensorrt_llm/llmapi/mm_encoder.py

```
@@ -0,0 +1,163 @@
+        use_tqdm: bool = True,
+    ) -> Union[RequestOutput, List[RequestOutput]]:
+        """Generate output for the given prompts in the synchronous mode.
+        Synchronous generation accepts either single prompt or batched prompts.
+
+        Args:
+            inputs (tensorrt_llm.inputs.data.PromptInputs, Sequence[tensorrt_llm.inputs.data.PromptInputs]): The prompt text or token ids.
+                It can be single prompt or batched prompts.
+        Returns:
+            Union[tensorrt_llm.llmapi.RequestOutput, List[tensorrt_llm.llmapi.RequestOutput]]: The output data of the completion request to the LLM.
+        """
+        unbatched = not isinstance(inputs, list)
```

Ruff (0.12.2)

1 blank line required between summary line and description

(D205)